samples/subsys/video/mega_full-featured/README.rst

```diff
@@ -0,0 +1,92 @@
+.. zephyr:code-sample:: video-mega_full-featured
+   :name: Arducam mega camera full-featured sample
+
+   Capture the video frames and transmit them to the PC software through
+   the serial port.
+
+Description
+***********
+
+This sample application caputre frames from a Arducam Mega camera and
+transmits them over the UART to the connected PC software, enabling camera
+control through the PC software.
+
+Requirements
+************
+
+This samples requires a Arducam Mega camera and uart support.
+
+- :ref:`rpi_pico`
+- `Arducam Mega camera module`_
+
+Wiring
+******
+
+On :ref:`rpi_pico`, The Arducam Mega camera module should be connected SPI0.
+The PC should be connected to UART0 using a USB cable.
+
+   .. note:: Be careful during connection!
+    Use separate wires to connect SPI pins with pins on the rpi_pico board.
```

Comment:
wrong indentation for the note block -- you may check rendered doc here: https://builds.zephyrproject.io/zephyr/pr/67538/docs/samples/subsys/video/mega_full-featured/README.html